src/main/java/Duke.java

```diff
@@ -1,10 +1,170 @@
-public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+import java.io.IOException;
+import java.util.Scanner;
+
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+
+
+public class Duke extends Application {
+    static String HOME = System.getProperty("user.home");
+    static java.nio.file.Path PATH = java.nio.file.Paths.get(HOME, "ip", "data.txt");
+
+    static Scanner sc = new Scanner(System.in);
+    static Ui ui = new Ui();
+    static Parser parser = new Parser();
+
+    static String s;
+    static TaskList tasklist;
```

Comment:
I feel like this can be either renamed `tasks` or `taskList` but I feel like this may be subjective.